src/platform_impl/ios/app_state.rs

```diff
@@ -525,8 +525,8 @@ pub fn did_finish_launching(mtm: MainThreadMarker) {
 
     // start waking up the event loop now!
     bug_assert!(
-        this.control_flow == ControlFlow::Poll,
-        "unexpectedly not setup to `Poll` on launch!"
+        this.control_flow == ControlFlow::default(),
+        "unexpectedly not setup to `ControlFlow::default()` on launch!"
```

Comment:
Not sure about this one as well.

Reply:
Moved to https://github.com/rust-windowing/winit/pull/3107, as it seems that this is an issue unrelated to this PR.